prow.sh

```diff
@@ -801,11 +801,12 @@ install_snapshot_controller() {
   fi
 
   cnt=0
-  expected_running_pods=$(curl https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/"${CSI_SNAPSHOTTER_VERSION}"/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml | grep replicas | cut -d ':' -f 2-)
-  while [ "$(kubectl get pods -l app=snapshot-controller | grep 'Running' -c)" -lt "$expected_running_pods" ]; do
+  expected_running_pods=$(awk -F '[: ]+' '$1 == "" && $2 == "replicas" {print $3; exit}' "${CONTROLLER_DIR}/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml")
```

Comment:
`CONTROLLER_DIR` is going to be a URL when testing something other than external-snapshotter. `awk` will fail in that case, won't it?

Reply:
Most possible yes, if target repository has no `setup-snapshot-controller.yaml` then it be failed.
We can add additional logic, to check if repository has this file, if not, it will be applied from the URL.